app/controllers/users_controller.rb

```diff
@@ -1,18 +1,16 @@
 class UsersController < ApplicationController
   before_action :logged_in_user, except: [:show, :new, :create]
-  before_action :load_user, except:[:index, :new, :create]
+  before_action :load_user, except: [:index, :new, :create]
   before_action :correct_user, only: [:edit, :update]
   before_action :admin_user, only: :destroy
 
   def index
-    @users = User.paginate page: params[:page],
+    @users = User.where(activated: true).paginate page: params[:page],
```

Comment:
Cái này chị nhớ em viết scope `where(activated: true)` rồi mà hì

giờ lại ko thấy, viết scope đi nhé

tìm hiểu về `scope in rails` nào